server/views/pages/recallEntry/check-sentences.njk

```diff
@@ -18,6 +19,12 @@
 {% endblock %}
 
 {% block content %}
+{% if crdError %}
```

Comment:
could consider moving this if statement and the `{% if errors.length %}` statement up into the prisonerBannerLayout.njk content block then calling `{{ super() }}`. There aren't many entry pages in recall but worth a consideration if it will ever get bigger